main.go

```diff
@@ -78,7 +79,10 @@ func main() {
 	var enableLeaderElection bool
 	var probeAddr string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", fmt.Sprintf(":%d", common.OperatorMetricsPort), "The address the metric endpoint binds to.")
+	var labelSelectorFlag string
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
```

Comment:
This seems to have got copied by mistake. This line is already present in the code

Reply:
`8080` port is hardcoded. Probably it got introduced when resolving merge conflicts.